src/validator.ts

```diff
@@ -10,13 +9,12 @@ import { StateMachine, StateMachineError } from './types';
 
 export default function validator(definition: StateMachine): {
   isValid: boolean;
-  errors: (Ajv.ErrorObject | StateMachineError)[];
+  errors: (ErrorObject | StateMachineError)[];
   errorsText: (separator?: string) => string;
 } {
 
   const errors = jsonSchemaErrors(definition);
   if (errors.length === 0) {
-    errors.push(...jsonPathErrors(definition));
```

Comment:
we don't need this step since the paths are validated as part of the `jsonSchemaErrors` call above